test/extractor/ConfigurationExtractor.test.ts

```diff
@@ -59,6 +59,33 @@ describe('ConfigurationExtractor', () => {
       expect(loggerSpy.getAllMessages('warn')).toHaveLength(0);
     });
 
+    it('should be able to handle a specified FHIR version', () => {
+      const ig = JSON.parse(
+        fs.readFileSync(path.join(__dirname, 'fixtures', 'simple-ig.json'), 'utf-8')
+      );
+      const result = ConfigurationExtractor.process([ig, ...resources], '5.0.0');
+      expect(result.config.fhirVersion).toEqual(['5.0.0']);
+    });
+
+    it('should be able to handle a specified FHIR version when FHIR version is missing', () => {
+      const ig = JSON.parse(
+        fs.readFileSync(path.join(__dirname, 'fixtures', 'missing-fhir-version-ig.json'), 'utf-8')
+      );
+      const result = ConfigurationExtractor.process([ig, ...resources], '5.0.0');
+      expect(result.config.fhirVersion).toEqual(['5.0.0']);
```

Comment:
Here I would also check the loggerSpy to confirm no errors/warnings were logged.